vars/gradlePipeline.groovy

```diff
@@ -0,0 +1,56 @@
+def call(Map pipelineParams) {
```

Comment:
Presumably this parameter is a placeholder for some later functionality?

Reply:
Yes you can pass parameters in. I am likely to use it again for the docker build. If not, I'll remove it then.